packages/polar/src/lib/createAccounts.ts

```diff
@@ -23,7 +24,8 @@ export async function createAccounts (n: number): Promise<Account[]> {
     accounts.push({
       name: "account_" + i.toString(),
       address: accAddress,
-      mnemonic: mnemonic
+      mnemonic: mnemonic,
+      signingPen: signingPen
```

Comment:
I don't think we need this in accounts, we can create signingPen using mnemonic.
note: we don't user to provide this, if user directly defines config in polar.config.js.